robohive/envs/arms/franka/assets/franka_ycb_v0.xml

```diff
@@ -10,12 +10,12 @@
 
     <size njmax='1000' nconmax='1000'/>
 
+
     <include file="../../../../simhive/scene_sim/topfloor_scene.xml"/>
     <include file="../../../../simhive/franka_sim/assets/assets.xml"/>
-    <include file="../../../../simhive/franka_sim/assets/gripper_assets.xml"/>
+    <include file="../../../../simhive/robotiq_sim/assets/assets.xml"/>
```

Comment:
Note that this is franka robot. Adding RobotiQ will cause confusion. We should create a new file for this and name it appropriately.